astro.config.mjs

```diff
@@ -27,9 +27,9 @@ export default defineConfig({
       [
         rehypeAutolinkHeadings,
         {
-          behavior: "prepend",
+          behavior: "append",
           content() {
-            return [h("span.heading-link", "#"), h("span.icon.icon-link")];
+            return [h("span.heading-link", "#")];
```

Comment:
I moved the # on the right on mobile, so it is always visible: 
![CleanShot 2024-03-21 at 10 56 38@2x](https://github.com/EuroPython/website/assets/667029/7d24eeca-fdaa-44b6-8d20-3349a0f17ce5)